articles/user-profile/normalized/auth0/index.md

```diff
@@ -2,6 +2,10 @@
 description: Auth0 normalizes common user properties in the User Profile.
 ---
 
+::: version-warning
+The user profile returned after successful authentication using an OIDC-compliant flow may differ from the Auth0 normalized profile. Please use the toggle to see the claims available on that profile.
+:::
+
```

Comment:
Re: Lock and auth0js - you currently can get the normalized profile (looks like the functionality exists because of specific client use case needs), but it's being deprecated in the near future, so I'm not going to mention it.